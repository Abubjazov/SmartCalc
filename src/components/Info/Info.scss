@import '../../variables';

.calc-info {
	align-self: start;
	padding: 20px;
	text-align: center;
	height: min-content;
	border-radius: 10px;
	box-shadow: 1px 1px rgba(238, 238, 238, 0.8);

	h3 {
		margin-top: 20px;
	}

	p {
		margin-top: 10px;
		font-size: 18px;
		font-weight: 300;
	}

	a {
		background-color: transparent;
	}

	.calc-info-clarification {
		margin-top: 30px;
		font-size: 14px;
		font-weight: 300;
	}
}
