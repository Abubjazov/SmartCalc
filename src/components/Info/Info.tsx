import './Info.scss'

export const Info = (): JSX.Element => (
	<div className='calc-info'>
		<h2>У QIWI всё под контролем</h2>
		<h3>Нас не коснулись никакие санкции ; )</h3>
		<p>Мы продолжаем проводить операции наших пользователей.</p>

		<h3>
			Когда срочно нужны деньги, пополните кошелек быстрым займом через наших
			партнеров.
		</h3>
		<p>
			Также деньги можно получить на банковскую карту или наличными в офисах
			CONTACT.
		</p>
		<p className='calc-info-clarification'>
			Услуга для лиц старше 18 лет. Понадобится паспорт РФ и почта. Иногда —
			телефон работодателя и авторизация в соцсети.
		</p>
	</div>
)
