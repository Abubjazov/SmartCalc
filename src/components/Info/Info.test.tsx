import { render } from '@testing-library/react'

import { Info } from './Info'

describe('Component: Spinner', () => {
	test('should render Spinners', () => {
		const { asFragment } = render(<Info />)

		expect(asFragment()).toMatchSnapshot()
	})
})
