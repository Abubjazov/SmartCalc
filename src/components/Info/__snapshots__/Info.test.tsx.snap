// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Spinner should render Spinners 1`] = `
<DocumentFragment>
  <div
    class="calc-info"
  >
    <h2>
      SMART CALC V.2
    </h2>
    <h3>
      Онлайн калькулятор
    </h3>
    <p>
      SPA на основе React, Redux и Typescript
    </p>
    <p>
      Репозиторий проекта:
    </p>
    <a
      href="https://github.com/Abubjazov/SmartCalc"
      rel="noopener noreferrer"
      target="_blank"
    >
      github.com/Abubjazov/SmartCalc
    </a>
    <p>
      Документация:
    </p>
    <a
      href="https://github.com/Abubjazov/SmartCalc/blob/main/README.md"
      rel="noopener noreferrer"
      target="_blank"
    >
      README.md
    </a>
    <p>
      Demo сервер:
    </p>
    <a
      href="https://abubjazov.github.io/SmartCalc/"
      rel="noopener noreferrer"
      target="_blank"
    >
      github.io/SmartCalc/
    </a>
    <p>
      Стэк технологий:
    </p>
    <h3>
      Typescript, React, ReactRouter, ReactHelmet, Redux, ReduxThunk, Axios, Nanoid, Bcryptjs, SASS
    </h3>
    <p
      class="calc-info-clarification"
    >
      Все права защищены! © Abubjazov 2022 (
      <a
        href="https://github.com/Abubjazov"
        rel="noopener noreferrer"
        target="_blank"
      >
        https://github.com/Abubjazov
      </a>
      )
    </p>
  </div>
</DocumentFragment>
`;
